.github/workflows/ci-integration-db-sqlalchemy-v1.yaml

```diff
@@ -0,0 +1,46 @@
+# CI - DB Integration - SqlAlchemy v1 is designed to run integration testing against SQLAlchemy v1.
```

Comment:
what is this file doing? it says target database clickhouse but we have more databases than that right? like oracle, Microsoft, etc

but I think those are tested on a different file. so I'm not sure about this file

Reply:
We had added clickhouse tests to the live (cloud workflow) because it uses SqlAlchemy v1 (it was not compatible with sqlalchemy 2). And we had decided not to create a separate one for only clickhouse. So I have just put the clickhouse part into a separate yaml.

Reply:
Reference discussion:
https://github.com/ploomber/jupysql/pull/810#issuecomment-1684016462
https://github.com/ploomber/jupysql/pull/810#issuecomment-1684034142